src/ucs/async/thread.c

```diff
@@ -53,27 +61,53 @@ static void ucs_async_thread_hold(ucs_async_thread_t *thread)
 static void ucs_async_thread_put(ucs_async_thread_t *thread)
 {
     if (ucs_atomic_fadd32(&thread->refcnt, -1) == 1) {
-        close(thread->epfd);
+        ucs_event_set_cleanup(thread->event_set);
         ucs_async_pipe_destroy(&thread->wakeup);
         ucs_timerq_cleanup(&thread->timerq);
         ucs_free(thread);
     }
 }
 
+static void ucs_async_thread_ev_handler(void *callback_data, int event,
+                                        void *arg)
+{
+    ucs_status_t status;
+    ucs_async_thread_callback_arg_t *cb_arg = (void*)arg;
+    int fd = (int)(uintptr_t)callback_data;
```

Comment:
pls, move variables with initialization at the begging of the function  and align by `=`

```
ucs_async_thread_callback_arg_t *cb_arg = (void*)arg;
int fd                                                                  = (int)(uintptr_t)callback_data
ucs_status_t status;
```

Reply:
done